src/transformers/models/nat/configuration_nat.py

```diff
@@ -65,7 +65,7 @@ class NatConfig(BackboneConfigMixin, PretrainedConfig):
             `"selu"` and `"gelu_new"` are supported.
         initializer_range (`float`, *optional*, defaults to 0.02):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
-        layer_norm_eps (`float`, *optional*, defaults to 1e-12):
+        layer_norm_eps (`float`, *optional*, defaults to 0.0):
```

Comment:
this is wrong should be `1e-05`

Reply:
Crap, it's the rounding of floats that does that. Will have a look when I can.